### DISCUSSION TITLE
Group Office as the default webmail

### DISCUSSION THREAD

**Discussion by lukaszhandy** [Intermesh/groupoffice#808] [General]
Hi
How can I add group office as my default mail handler (via google chome).
It migrates from SOGo, which enables such functionality and it is very useful.
In the chrome: // settings / handlers - protocol handling section, it is not possible to select group office as the default e-mail client.
Greetings

**Comment by migoe101** [comment]
As my understanding and use is to licence pro, install Group Office Assistent and after that you can set GO Assistant as default Mail Client in all Desktop Applications inkl. Chrome.

**Comment by mschering** [comment]
This part will also work without pro I think.
